ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -167,7 +167,7 @@ func createCluster(context *cli.Context, rdwr config.ReadWriter, ecsClient ecscl
 
 	// Check if cfn stack already exists
 	cfnClient.Initialize(ecsParams)
-	stackName := ecsParams.GetCFNStackName()
```

Comment:
Can we still read from the old ini config?

Reply:
Yeah, the CFN stack name change works. There's an issue with the Project Name prefix, but I'm fixing that.